### PR TITLE
Fixes of i18n on get-and-update-the-seller-exchange-window.md

### PR DESCRIPTION
**Pull request by GeorgeLimaDev**
It relates to task [LOC-10435](https://vtex-dev.atlassian.net/browse/LOC-10435) and KR 23Q2-KR1 ([Review all developer portal documentation in English](https://docs.google.com/document/d/1VduAYpjqdazhyGH5DvD9c52pMu1Xfj4RDx48QNjTFU0/edit)).

#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [X] Spelling and grammar accuracy (self-explanatory)

[LOC-10435]: https://vtex-dev.atlassian.net/browse/LOC-10435?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 1]
```diff
@@ -2,36 +2,35 @@
...
 
+This guide describes how to get a seller change window or update the number of days in a marketplace on your account.
 
-This guide describes how to get a seller's current exchange window or update this value in a marketplace on your account.
+## Getting the current seller change window
 
-## Getting the current window to change seller
-
-To get the window to change seller in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` through the URL address, as shown by the example below:
+To get seller change window in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` in the URL address, as in the example below:
```
Comment (ricardoaerobr) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:19]: Atualizar link: [Get window to change seller](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pvt/configuration/window-to-change-seller)

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 2]
```diff
@@ -2,36 +2,35 @@
...
 
 ```json
 "4"
 ```
 
-## Updating the window to change seller
+## Updating the window to change sellers
 
-To update the seller's exchange window (in days), you need to use a similar request of the **Get window to change seller**, but as a **POST** request ([Update window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller)), as shown by the example below:
+To update the seller change window (in days), you need to use a request similar to **Get window to change seller**, but as a **POST** request ([Update window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller)), as in the example below:
```
Comment (ricardoaerobr) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:33]: Atualizar link: [Update window to change seller](https://developers.vtex.com/docs/api-reference/orders-api#post-/api/checkout/pvt/configuration/window-to-change-seller)

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 3]
```diff
@@ -43,18 +42,18 @@ See a request body example below (10 days):
...
     }
-```    
-    
-After sending the request, the endpoint will return `code 201 (Created)` and an empty response body. 
+```
+
+After sending the request, the endpoint will return `code 201 (Created)` and an empty response body.
 
-To confirm that the new window to change seller have been applied to your account, access the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint again as a **GET** request.
+To confirm that the new seller change window has been applied to your account, access the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint again as a **GET** request.
```
Comment (ricardoaerobr) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:49]: Atualizar link: [Get window to change seller](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pvt/configuration/window-to-change-seller)

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 4]
```diff
@@ -2,36 +2,35 @@
...
 
-In this situation, VTEX allows the marketplace to have a period of time (window), after the customer completes the purchase, to change sellers and thus avoid an order cancellation. For more information on the process of switching sellers, access the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+In such situations, VTEX allows the marketplace to have a period (window) after the customer makes the purchase to change sellers and avoid order cancellation. For more information on the process of changing sellers, please read the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+> ⚠️ The default window for changing a seller is <strong>2 days</strong>. However, you can change it for up to <strong>30 days</strong>.
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:12]:
```suggestion

> ⚠️ The default window for changing a seller is **2 days**. However, you can change it for up to **30 days**.
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 5]
```diff
@@ -70,11 +69,11 @@ The following errors may appear as a message in the response body.
 
 ### 404 - Not Found
 
-- **Message error example**: `"The requested URL was not found on the server"`: check that the URL data is correct.
+- **Message error example**: `"The requested URL was not found on the server"`. Check if the URL is correct.
 
 ```json
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:74]:
```suggestion
```html
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 6]
```diff
@@ -2,36 +2,36 @@
 title: "Get and update the seller exchange window"
 slug: "get-and-update-the-seller-exchange-window"
 hidden: true
-createdAt: "2022-12-12T21:19:51.360Z"
+createdAt: "2022-12-12t21:19:51.360z"
 updatedAt: "2022-12-12T21:36:31.439Z"
 ---
 
-[Marketplace](https://help.vtex.com/en/tutorial/integrating-with-marketplace) is the platform where products from different sellers can be offered to customers in one place. After the customer completes a purchase and the order is created, there may be scenarios in which the original seller of the product needs to be modified, for example, when their cancels the order due to an internal unavailability of product fulfillment.
+[Marketplace](https://help.vtex.com/en/tutorial/integrating-with-marketplace) is a platform where different sellers can sell their products in a single place. After the customer makes a purchase and the order is created, there may be scenarios in which the original seller of the product needs to be modified, for example, when they cancel the order due to internal unavailability of product fulfillment.
```
Comment (julia-rabello) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:9]:
```suggestion
A [Marketplace](https://help.vtex.com/en/tutorial/integrating-with-marketplace) is a platform where products from different sellers can be offered to customers in a single place. After the customer makes a purchase and the order is created, there may be scenarios in which the original seller of the product needs to be modified, for example, when they cancel the order due to internal unavailability of product fulfillment.
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 7]
```diff
@@ -2,36 +2,36 @@
...
 
-In this situation, VTEX allows the marketplace to have a period of time (window), after the customer completes the purchase, to change sellers and thus avoid an order cancellation. For more information on the process of switching sellers, access the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+In such situations, VTEX allows the marketplace to have a period (window) after the customer makes the purchase to change sellers and avoid order cancellation. For more information on the process of changing sellers, please read the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
```
Comment (julia-rabello) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:11]:
```suggestion
In such situations, after the customer makes the purchase, VTEX allows the marketplace to have a period of time (window) to change sellers and avoid order cancellation. For more information on the process of changing sellers, please read the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md [hunk 8]
```diff
@@ -2,36 +2,36 @@
...
 
->⚠️ The default window for the change seller is <strong>2 days</strong>. However, you can modify the value for up to a maximum period of <strong>30 days</strong>.
+> ⚠️ The default window for changing a seller is **2 days**. However, you can change it for up to **30 days**.
```
Comment (julia-rabello) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md:13]:
```suggestion
> ⚠️ The default window for changing a seller is **2 days**. However, you can set the window for up to **30 days**.
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**